upgrade/upd_2.5.11-to-2.7.0/index.php

```
@@ -607,7 +613,164 @@ public function apply_deleteflashsanitizer(): bool
+
+    /**
+     * Check if the `sort` index already uses a 100-char prefix on step_name.
+     *
+     * @return bool true if already normalised (no action needed)
+     */
+    public function check_normalizeprofileregstepsort(): bool
+    {
+        $table = $this->db->prefix('profile_regstep');
+
+        return 100 === $this->indexPrefixLength($table, 'sort', 'step_name');
+    }
```

PHPMD (2.15.0)

[error] The method check_normalizeprofileregstepsort is not named in camelCase. (undefined)

(CamelCaseMethodName)